src/main/java/HomePage.java

```diff
@@ -0,0 +1,39 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class HomePage {
+    private final WebDriver driver;
+    private final String EXPECTED_NAME = "Inbox — Yandex.Mail";
+    private final By userAccountName = By.xpath("//div[@class='legouser__menu-header']//span[@class='user-account__name']");
+    private final By userIcon = By.className("user-pic__image");
+    private final By signOutButton = By.linkText("Log out");
+
+    public HomePage(WebDriver driver) {
```

Comment:
No needs to pass WebDriver instance as param

Reply:
This was changed